rust/kcl-lib/std/sketch.kcl

```diff
@@ -33,10 +33,12 @@ export fn circle(
   /// Sketch to extend, or plane or surface to sketch on.
   @sketch_or_surface: Sketch | Plane | Face,
   /// The center of the circle.
+  @(snippet_array = ["0", "0"])
```

Comment:
I think we've been using camel case rather than snake for attributes, following the KCL variable convention

Reply:
Ah, thanks. I was confused because I saw `@(impl = std_rust)` and thought it was snake case -- we can always change that one too though.